.travis.yml

```diff
@@ -22,7 +22,7 @@ before_script:
 
 install: true
 
-jobs:  
+jobs:
   include:
     - stage: 'Code Quality'
```

Comment:
Please divide it into 2 stage: the first one is only for code quality analysis, and the second one for unit test (credo). So we dont have to run our test if code quality analysis is not passed